cli/dts/lib.deno.ns.d.ts

```diff
@@ -710,11 +710,11 @@ declare namespace Deno {
   }
 
   /** A handle for `stdin`. */
-  export const stdin: Reader & ReaderSync & Closer & { rid: number };
+  export const stdin: Reader & ReaderSync & Closer & { readonly rid: number };
   /** A handle for `stdout`. */
-  export const stdout: Writer & WriterSync & Closer & { rid: number };
+  export const stdout: Writer & WriterSync & Closer & { readonly rid: number };
   /** A handle for `stderr`. */
-  export const stderr: Writer & WriterSync & Closer & { rid: number };
+  export const stderr: Writer & WriterSync & Closer & { readonly rid: number };
```

Comment:
This is breaking change 😬

Reply:
Technically, yes but I don't think we ever intended for them to actually be mutable.

Reply:
@caspervonb that's true, I'm only afraid it's gonna cause type checking havoc, hopefully not many users access `rid` of stdio objects.

Reply:
Access should be fine - it should only break if someone tries to update it.